Sources/PackagePlugin/CMakeLists.txt

```diff
@@ -7,60 +7,63 @@
 # See http://swift.org/CONTRIBUTORS.txt for Swift project authors
 
 add_library(PackagePlugin
-    PublicAPI/CommandConstructor.swift
```

Comment:
While modifying the CMakefile I made it conform to the 2-space indentation that we seem to be using for CMakefiles.